bing_webmaster_tools/models/url_management.py

```diff
@@ -11,3 +12,12 @@ class QueryParameter(BingModel):
     is_enabled: bool
     parameter: str
     source: int
+
+
+QueryParamStr = Annotated[
+    str,
+    StringConstraints(
+        min_length=1,
+        pattern=r"^[a-zA-Z0-9:]+$",  # Only alphanumeric and colon
```

Comment:
Query param keys can often contain hyphens and underscore, such as `utm_campaign=` or `item-color=`. Do we need to add those two characters here?